helm-charts/whitesource-renovate/templates/deployment.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "whitesource-renovate.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "whitesource-renovate.name" . }}
+    helm.sh/chart: {{ include "whitesource-renovate.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "whitesource-renovate.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "whitesource-renovate.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            {{- if .Values.renovate.acceptWhiteSourceTos }}
+            - name: ACCEPT_WHITESOURCE_TOS
+              value: {{ .Values.renovate.acceptWhiteSourceTos | quote }}
+            {{- end }}
+            {{- if .Values.renovate.licenseKey }}
+            - name: LICENSE_KEY
+              valueFrom:
+                secretKeyRef:
+                  secretName: {{ .Release.Name }}
```

Comment:
I think for current version of k8s this should be `name`. At least this fails on our `v1.17.6` cluster. There are a couple more places where this needs to be replaced.

Reply:
Confirmed in the doc, it should be `name` when using `secretKeyRef`. Updated the manifests to fix that!